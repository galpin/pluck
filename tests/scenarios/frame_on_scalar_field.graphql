{
  cores(limit: 5) {
    id @frame
    status
    missions {
      name
      flight
    }
  }
}
