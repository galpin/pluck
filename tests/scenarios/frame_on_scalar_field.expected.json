{
 "id": [
  {
   "id": "B1015"
  },
  {
   "id": "B0006"
  },
  {
   "id": "B1034"
  },
  {
   "id": "B1016"
  },
  {
   "id": "B1025"
  }
 ]
}